dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/AssemblyCheckFilterProcessorTest.java

```diff
@@ -0,0 +1,197 @@
+
+        SubSnpCoreFields processedDeletion = assemblyChecker.process(deletionWithInvalidReference);
+        assertNull(processedDeletion);
+    }
+
+    
+    @Test
+    public void validMNV() throws Exception {
+        SubSnpCoreFields mnvWithValidReference = new SubSnpCoreFields(317288161L, Orientation.FORWARD, 431853804L,
+                                                                      Orientation.FORWARD, "NT_455997.1", 1340260L,
+                                                                      1340261L, Orientation.REVERSE,
+                                                                      LocusType.SHORTER_ON_CONTIG, "22", 2532L, 2533L,
+                                                                      "TC", "TC", "A", "A/C",
+                                                                      "NC_006109.4:g.2532_2533delGAinsT", 2532L, 2533L,
+                                                                      Orientation.REVERSE,
+                                                                      "NT_455997.1:g.1340260_1340261delTCinsA",
+                                                                      1340260L, 1340261L, Orientation.FORWARD,
+                                                                      "1055116");
+
+        SubSnpCoreFields processedMnv = assemblyChecker.process(mnvWithValidReference);
+        assertNotNull(processedMnv);
+    }
+
+    @Test
+    public void invalidMNV() throws Exception {
+        SubSnpCoreFields mnvWithInvalidReference = new SubSnpCoreFields(317288161L, Orientation.FORWARD, 431853804L,
+                                                                      Orientation.FORWARD, "NT_455997.1", 1340260L,
+                                                                      1340261L, Orientation.REVERSE,
+                                                                      LocusType.SHORTER_ON_CONTIG, "22", 2532L, 2533L,
+                                                                      "TT", "TT", "A", "A/C",
+                                                                      "NC_006109.4:g.2532_2533delGGinsT", 2532L, 2533L,
+                                                                      Orientation.REVERSE,
+                                                                      "NT_455997.1:g.1340260_1340261delTTinsA",
+                                                                      1340260L, 1340261L, Orientation.FORWARD,
+                                                                      "1055116");
+
+        SubSnpCoreFields processedMnv = assemblyChecker.process(mnvWithInvalidReference);
+        assertNull(processedMnv);
+    }
+
+    @Test
+    public void validSnpWithNoChromosomeCoordinates() throws Exception {
+        // we have replaced the Refseq contig for an Ensembl one, but not in hgvsTString, because this is what the class
+        // "RefseqToGenbankMappingProcessor" does
```

Comment:
I don't understand what this comment means: `RefseqToGenbankMappingProcessor` does not change `hgvsTString`.